src/java/org/apache/cassandra/metrics/CassandraMetricsRegistry.java

```diff
@@ -41,15 +57,20 @@
  */
 public class CassandraMetricsRegistry extends MetricRegistry
 {
-    public static final CassandraMetricsRegistry Metrics = new CassandraMetricsRegistry();
+    private static final Logger logger = LoggerFactory.getLogger(CassandraMetricsRegistry.class);
+
+    public static final CassandraMetricsRegistry Metrics = new CassandraMetricsRegistry(Ints.checkedCast(TimeUnit.MICROSECONDS.toNanos(1)));
     private final Map<String, ThreadPoolMetrics> threadPoolMetrics = new ConcurrentHashMap<>();
 
     private final MBeanWrapper mBeanServer = MBeanWrapper.instance;
     public final static TimeUnit DEFAULT_TIMER_UNIT = TimeUnit.MICROSECONDS;
 
-    private CassandraMetricsRegistry()
+    final ScheduledFuture<?> periodicMeterTicker;
+
+    CassandraMetricsRegistry(int tickMetersPeriodMicros)
```

Comment:
I think we don't need to pass the argument to the constructor and can use a constant variable instead. We can also add a javadoc to this constant so that the default value can be indexed by search engines once the javadocs are published, 

e.g. 
`/* Schedule a task to read all the meters once a day so it isn’t done in the request path and we have a more incremental amount to process at a time. Default value {@code 1} day. */ `

Reply:
This is to make it possible for the unit test to create a metrics registry that runs these ticks frequently. To have it be testable there has to be some extension point, either a method that can be overridden or a parameter to the class to allow for manipulating the task scheduling frequency.